CONTRIBUTING.md

```diff
@@ -34,13 +35,13 @@ Change the client configuration in the example to look like this:
 
 ```tsx
 // ...
-import { createClient, fetchExchange, cacheExchange } from "urql";
+import { createClient, defaultExchanges } from "urql";
 import { devtoolsExchange } from "<path-to-devtools-dist>/exchange";
 
 // ...
 const client = createClient({
   url: "http://localhost:3001/graphql",
-  exchanges: [devtoolsExchange, cacheExchange, fetchExchange]
+  exchanges: [devtoolsExchange, ...defaultExchanges]
```

Comment:
I'm hesitant to do this as it doesn't actually represent how the exchanges are working in practice.

With this change, operations are being deduped but only after they have already been seen by the devtools exchange. This means the operations displayed in devtools and the real world operations being executed are different.

Because  exchanges are order dependent, the correct way to use the default exchanges with devtools is like this:

```
exchanges: [dedupExchange, devtoolsExchange, cacheExchange, fetchExchange],
```

Reply:
All right, I see what you mean. This makes sense, prefer me adding the `dedupExchange`? Since this is included in our defaults?

Reply:
100% go for it!

Reply:
to be fair, both of these are going to work and we have to trust the users to understand the difference or explain it somewhere (there are notes on this in the new guides section)

but let's make sure that we integrate these changes first maybe and add the other ones on top? https://github.com/FormidableLabs/urql-devtools/pull/41/files#diff-04c6e90faac2675aa89e2176d2eec7d8R32

Reply:
All right, I've adjusted them